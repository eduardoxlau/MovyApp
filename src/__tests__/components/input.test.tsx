/* eslint-disable no-unused-vars */
/* eslint-disable no-undef */
import { render } from '@testing-library/react';

import Input from 'components/input';

describe('<Input />', () => {
  it('should render snapshot input light', () => {
    const { asFragment } = render(
      <Input theme="light" onChange={() => null} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render snapshot input dark', () => {
    const { asFragment } = render(<Input theme="dark" onChange={() => null} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should required input', () => {
    render(<Input required onChange={() => null} />);
    expect(document.querySelector('input')).toBeRequired();
  });
});
