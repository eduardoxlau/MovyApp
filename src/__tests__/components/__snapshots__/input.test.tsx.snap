// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should render snapshot input dark 1`] = `
<DocumentFragment>
  <input
    class="rounded w-full h-10 my-1.5 text-indent bg-black border border-button indent-1"
    name=""
    placeholder=""
    type="text"
    value=""
  />
</DocumentFragment>
`;

exports[`<Input /> should render snapshot input light 1`] = `
<DocumentFragment>
  <input
    class="rounded w-full h-10 my-1.5 text-black indent-1"
    name=""
    placeholder=""
    type="text"
    value=""
  />
</DocumentFragment>
`;
